tests/xharness/Program.cs

```diff
@@ -90,16 +115,44 @@ public static int Main (string [] args)
 			var input = os.Parse (args);
 			if (input.Count > 0)
 				throw new Exception (string.Format ("Unknown arguments: {0}", string.Join (", ", input.ToArray ())));
-			if (harness.Action == HarnessAction.None)
+			
+			if (action == HarnessAction.None)
 				showHelp ();
```

Comment:
@rolfbjarne should we `return 1` here?

Reply:
Yes, I think that's fine.

Reply:
Ah, `showHelp` already calls `Environment.Exti(0)`..